spacewalk/uyuni-setup-reportdb/bin/uyuni-setup-reportdb

```diff
@@ -355,6 +370,20 @@ exists_schema() {
     return $?
 }
 
+get_rhnconf_value() {
+   VAL1=$(egrep "^$1[[:space:]]*=" $RHN_CONF)
+   VAL2=$(echo $VAL1 | sed "s/^$1[[:space:]]*=[[:space:]]*\(.\+\)/\1/" 2>/dev/null)
+   echo $VAL2
+}
+
+rhn_reconfig() {
+    if egrep "^$1[[:space:]]*=" $RHN_CONF >/dev/null; then
```

Comment:
Very minor thing, but `egrep` is kind of depracated. `grep -E` should be used instead. From the man page:

> In addition, the variant programs egrep and fgrep are the same as grep -E and grep -F, respectively.  These variants  are  deprecated,  but are provided for backward compatibility.